src/main/java/com/erii/ui/ControlPanel.java

```diff
@@ -0,0 +1,153 @@
+
+        System.out.println("How may I assist you today?");
+        System.out.println("1. List tasks");
+        System.out.println("2. Add a task");
+        System.out.println("3. Add a deadline task");
+        System.out.println("4. Add an event task");
+        System.out.println("5. Mark a task as done");
+        System.out.println("6. Exit");
+        System.out.print("Enter the number corresponding to your choice: ");
+
+        while (scanner.hasNextLine()) {
+            String choice = scanner.nextLine().trim();
+
+            switch (choice) {
+                case "1":
+                    listTasks(taskManager);
+                    break;
+                case "2":
+                    System.out.println("Please enter the task description and priority (e.g., slain a dragon /S):");
+                    String inputAddTask = scanner.nextLine().trim();
+                    addTodoTask(inputAddTask, taskManager);
+                    break;
+                case "3":
+                    System.out.println("Please enter the deadline task description, deadline date and priority (e.g., submit report /by 2021-09-30 /SS):");
+                    String inputAddDeadline = scanner.nextLine().trim();
+                    addDeadlineTask(inputAddDeadline, taskManager);
+                    break;
+                case "4":
+                    System.out.println("Please enter the event description, start date, end date and priority (e.g., project meeting /from 2021-09-30 /to 2021-10-01 /S):");
+                    String inputAddEvent = scanner.nextLine().trim();
+                    addEventTask(inputAddEvent, taskManager);
+                    break;
+                case "5":
+                    System.out.println("Please enter the task number to mark as done:");
+                    String inputMark = scanner.nextLine().trim();
+                    markTaskAsDone(inputMark, taskManager);
+                    break;
+                // ...
+
+                case "6":
+                    String address = getAddress();
+                    System.out.println("Pleased to serve you, " + address + "." + UserDetails.getUserName());
+                    break;
+                default:
+                    System.out.println("Unknown command. Please try again.");
+                    break;
+            }
+            System.out.println("Enter next command:");
+        }
+        scanner.close();
+    }
+    
+    private static void listTasks(TaskManager taskManager) {
+        taskManager.listTasks();
+    }
+    
+    private static void addTodoTask(String input, TaskManager taskManager) {
+        // Splitting the input at " /" to separate description and priority
+        String[] parts = input.split(" ?/ ?");
+        if (parts.length < 2 || parts[1].isEmpty()) {
+            System.out.println("\"Incorrect format. Please ensure the task description is followed by '/' and a priority value (e.g., 'slain a dragon /SS').\"");
```

Comment:
Line length should be no longer than 120 chars. line contains 149 characters, consider splitting it into two lines 

https://se-education.org/guides/conventions/java/basic.html#layout